src/components/core/pivotTable/agGridUtils.ts

```diff
@@ -258,3 +258,7 @@ export const sanitizeFingerprint = (fingerprint: string): string => {
         afm: sanitizedParsedFingerprintAfm,
     });
 };
+
+export const isColumnDisplayed = (displayedColumns: Column[], column: Column) => {
+    return displayedColumns.find(displayedColumn => displayedColumn.getColId() === column.getColId());
```

Comment:
replace `.find` with `.some()` as we need only a boolean value?

Reply:
fixed